polus-imagej-filter-convolve-plugin/README.md

```diff
@@ -0,0 +1,30 @@
+# ImageJ filter convolve
+
+This WIPP plugin was automatically generated by a utility that searches for
+ImageJ plugins and generates code to run them. For more information on what this
+plugin does, contact one of the authors: Nick Schaub (nick.schaub@nih.gov) or 
+Anjali Taneja (anjali.taneja@axleinfo.com).
+
+For more information on WIPP, visit the [official WIPP page](https://isg.nist.gov/deepzoomweb/software/wipp).
+
+## Building
+
+To build the Docker image for the conversion plugin, run
+`./build-docker.sh`.
+
+## Install WIPP Plugin
+
+If WIPP is running, navigate to the plugins page and add a new plugin.
+Paste the contents of `plugin.json` into the pop-up window and submit.
+
+## Options
+
+This plugin takes one input argument and one output argument:
+
+| Name          | Description             | I/O    | Type   |
+|---------------|-------------------------|--------|--------|
+| `--inpDir` | inpDir | Input | collection |
+| `--weights` | weights | Input | collection |
```

Comment:
What do these weights mean?

Reply:
These weights represent a filter kernel which is passed a collection from WIPP.